components/atom/button/test/index.test.js

```diff
@@ -542,15 +542,16 @@ describe(json.name, () => {
   })
 
   describe('atomButtonColors', () => {
```

Comment:
check the other exported object tests and copy them

Reply:
solved